xml/System.Text/Encoding.xml

```diff
@@ -3902,7 +3902,9 @@ In .NET 5.0 and later versions, the code page name `utf-7` is not supported.
  This method returns a list of supported encodings, uniquely distinguished by code page. For a table that lists the supported encodings, see <xref:System.Text.Encoding>.
 
 > [!NOTE]
-> The list of supported encodings returned by the <xref:System.Text.Encoding.GetEncodings%2A> method does not include any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calls to the <xref:System.Text.Encoding.RegisterProvider%2A> method.
+> In .NET 5.0 and later versions, the list of supported encodings that <xref:System.Text.Encoding.GetEncodings%2A> returns includes any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calling <xref:System.Text.Encoding.RegisterProvider%2A>.
+
+In .NET Framework and .NET Core, the returned list of supported encodings *does not* include these additional encodings.
```

Comment:
by the way, is it usual to use * in the text as what we have around `does not`? or these will be rendered differently in the page?

Reply:
It renders as italics - see https://review.docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getencodings?view=net-5.0&branch=pr-en-us-4821#remarks.